app/src/main/java/woowacourse/shopping/data/repository/DefaultCartRepository.kt

```
@@ -81,6 +85,8 @@ class DefaultCartRepository(
             val cartContent = getCartContents().find { it.product.id == productId } ?: return RemoveItemResult.NotFoundItem
             remoteDataSource.deleteItem(cartContent.id)
             RemoveItemResult.Success(getCart())
+        } catch (err: CancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)